ledger/accountdb.go

```diff
@@ -2334,6 +2364,14 @@ func (w *accountsSQLWriter) close() {
 		w.updateResourceStmt.Close()
 		w.updateResourceStmt = nil
 	}
+	if w.deleteKvPairStmt != nil {
```

Comment:
@jannotti I'm still working to understand `ledger/accountdb.go` though it struck me the following _optional_ refactoring may be seen as beneficial:  https://github.com/jannotti/go-algorand/pull/10.

Can you have a look?  Feel welcomed to close the PR + resolve thread if you prefer as is.

Reply:
I added this by hand, as the PR had fallen into a deep abyss of conflicts. Thanks.

Reply:
@jannotti Thanks for bringing in the change and dealing with the conflicts.